### DISCUSSION TITLE
PET parameters

### DISCUSSION THREAD

**Discussion by Soossee** [UMEP-dev/UMEP#491] [General]
Hi everyone,
I'm doing some experiments with SOLWEIG and I would like to ask if anyone can give me any indication regarding PET parameters. In particular, can anyone tell me how acitivity (W) and Clothing (clo) work?
Any guides or links would be extremely helpful to me, thank you!!

**Comment by nilswallenberg** [comment]
Hi,

Two papers on PET and its applications can be found here: [Höppe (1999)](https://doi.org/10.1007/s004840050118) and [Matzarakis et al. (1999)](https://doi.org/10.1007/s004840050119)

W (Watt) is the activity at hand, i.e. if a person is standing still, running around, or something else. Default value (80 W) corresponds to light activity, i.e. standing person. clo (0.9 default, corresponding to pants, shoes, shirt I think. There are ISOs on this topic.) is the clothing, where different clothing have different heat resistance. Higher clo equals more clothing and vice versa.

PET, as far as I understand, is not very sensitive to different W or clo. Tmrt, on the other hand, have a large effect on PET (e.g. [Mayer et al. (2008)](https://www.researchgate.net/publication/228626632_Human_thermal_comfort_in_summer_within_an_urban_street_canyon_in_Central_Europe) and [Lee et al. (2013)](https://doi.org/10.1155/2013/312572)). 

Cheers,
Nils
